test cases/d/11 dub/meson.build

```diff
@@ -1,5 +1,7 @@
 project('dub-example', 'd')
 
+error('MESON_SKIP_TEST: Dub support is broken at the moment (#11773)')
```

Comment:
This should not have passed review and been merged. Don't unconditionally error out -- that means this PR doesn't have any testsuite coverage at all, even on CI runners with older versions of Dub installed.

This is the only test that even attempts to do `import('dlang')`, let alone use the result. We have no idea if even that works!!